<?xml version="1.0"?>
<launch>
    <node name="pub_saveflag" pkg="sensor_fusion" type="pub_saveflag" output="screen">
        <remap from="/flag" to="/save_flag" />
    </node>
</launch>
